kinetic/distribution.yaml

```diff
@@ -427,6 +427,35 @@ repositories:
       url: https://github.com/iirob/ati_force_torque.git
       version: kinetic-devel
     status: developed
+  aubo_robot:
+    doc:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: kinetic-devel
+    release:
+      packages:
+      - aubo_control
+      - aubo_description
+      - aubo_driver
+      - aubo_gazebo
+      - aubo_i5_moveit_config
+      - aubo_kinematics
+      - aubo_msgs
+      - aubo_new_driver
+      - aubo_panel
+      - aubo_robot
+      - aubo_trajectory
+      - aubo_trajectory_filters
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/auboliuxin/aubo_robot-release.git
+      version: 0.3.18-0
+    source:
+      test_pull_requests: true
```

Comment:
It doesn't look like that you have followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user. Please either remove this line or provide the necessary access as described in the wiki page.